README.md

```diff
@@ -332,6 +328,48 @@ The modality provides two options:
+| Mint            | recipient (Key), token_id (String)                                      |
+| Transfer        | owner (Key), operator (Option<Key>), recipient (Key), token_id (String) |
+| Burn            | owner (Key), token_id (String)                                          |
+| ApprovalGranted | owner (Key), spender (Key), token_id (String)                           |
+| ApprovalRevoked | owner (Key), token_id (String)                                          |
+| ApprovalForAll  | owner (Key), operator (Key)                                             |
+| RevokedForAll   | owner (Key), operator (Key)                                             |
+| MetadataUpdate  | token_id (String)                                                       |
+| Migration       | -                                                                       |
+| VariablesSet    | -                                                                       |
+
+##### Casper Event Standard
+
+`CES` is an option within the `EventsMode` modality that determines how changes to tokens issued by the contract instance will be recorded. Any changes are recorded in the `__events` dictionary and can be observed via a node's Server Side Events stream. They may also be viewed by querying the dictionary at any time using the JSON-RPC interface.
+
+The emitted events are encoded according to the [Casper Event Standard](https://github.com/make-software/casper-event-standard), and the schema is visible to an observer reading the `__events_schema` contract named key.
+
+For this CEP-78 reference implementation, the events schema is as follows:
+
+| Event name      | Included values and type                                                |
+| --------------- | ----------------------------------------------------------------------- |
+| Mint            | recipient (Key), token_id (String), data (String)                       |
+| Transfer        | owner (Key), operator (Option<Key>), recipient (Key), token_id (String) |
+| Burn            | owner (Key), token_id (String)                                          |
+| Approval        | owner (Key), spender (Key), token_id (String)                           |
+| ApprovalRevoked | owner (Key), token_id (String)                                          |
+| ApprovalForAll  | owner (Key), operator (Key)                                             |
+| RevokedForAll   | owner (Key), operator (Key)                                             |
+| MetadataUpdated | token_id (String), data (String)                                        |
+| Migration       | -                                                                       |
+| VariablesSet    | -                                                                       |
+
+#### Modality Conflicts
+
+The `MetadataMutability` option of `Mutable` cannot be used in conjunction with `NFTIdentifierMode` modality of `Hash`.
```

Comment:
This would be easier to understand:
```suggestion
The `MetadataMutability` option set to `Mutable` cannot be used in conjunction with the `NFTIdentifierMode` modality set to `Hash`.
```